VideoLocker/src/main/java/org/edx/mobile/module/notification/NotificationPreference.java

```diff
@@ -0,0 +1,85 @@
+package org.edx.mobile.module.notification;
+
+import org.edx.mobile.model.api.CourseEntry;
+
+import java.io.Serializable;
+import java.util.ArrayList;
+import java.util.HashSet;
+import java.util.LinkedList;
+import java.util.List;
+import java.util.Set;
+
+/**
+ * We extends HashMap to make the implementation extendible
+ */
+public class NotificationPreference extends ArrayList<EdxLocalParseChannel> implements Serializable{
```

Comment:
It might be better if this had an array list as a member than was an array list.